### PR TITLE
Add request for high table

### PR DESCRIPTION
**Pull request by rgaiacs**
@lexnederbragt Could you review this?

### CODE REVIEW

File: pages/workshops_operations.html [hunk 1]
```diff
@@ -481,7 +481,7 @@ <h3 id="host-registration">Registration</h3>
   the registration page further.
 </p>
 
-<h3 id="book-room">The Workshop Venue</h3>
+<h3 id="host_venue">The Workshop Venue</h3>
```
Comment (rgaiacs) [inline pages/workshops_operations.html:484]:
```
$ grep -r book-room pages 
pages/workshops_operations.html:<h3 id="book-room">The Workshop Venue</h3>
$ grep -r host_venue pages
pages/checklists_host.html:      <a href="{{site.baseurl}}/workshops/operations/#host_venue">good venue</a> available?
pages/checklists_host.html:      the <a href="{{site.baseurl}}/workshops/operations/#host_venue">ideal room characteristics</a>.
```

File: pages/workshops_operations.html [hunk 2]
```diff
@@ -527,6 +527,12 @@ <h3 id="book-room">The Workshop Venue</h3>
   </li>
   <li>
     <p>
+      <strong>High table</strong>
+      to allow instructors be stand when live coding.
```
Comment (lexnederbragt) [inline pages/workshops_operations.html]: or one that can be raised, to allow instructors to stand when live coding.

**Comment by lexnederbragt** [comment]
I made one wording suggestion, otherwise +1

**Comment by jduckles** [comment]
Isn't this just a podium?

**Comment by lexnederbragt** [comment]
Wikipedia says "Common parlance has shown an increasing use of podium in American English to describe a lectern." Still, I am worried non-English speakers may interpret 'podium' as an elevation they stand on.

How about

"a lectern, desk or table that can be raised or a high table"
 "--to allow instructors to stand when live coding."

**Comment by jduckles** [comment]
@lexnederbragt, that works for me.

**Comment by jduckles** [comment]
@rgaiacs can you use the language that @lexnederbragt suggested and then I'll merge.

**Comment by gvwilson** [comment]
@rgaiacs please amend and then we'll merge.

**Comment by rgaiacs** [comment]
@lexnederbragt Thanks for the suggestion.

@gvwilson Sorry for the delay.

**Pull request merged**